daemon/cmd/status.go

```diff
@@ -6,6 +6,7 @@ package cmd
 import (
 	"context"
 	"fmt"
+	rand "math/rand/v2"
```

Comment:
nit: the alias here is redundant

Reply:
Yes, I'm aware.  I'm not sure we have any coding style guidelines regarding named imports. Personally I prefer to add an explicit alias in case the base name of the import (i.e. `v2` in this case) doesn't match the package name. But that might be redundant for versioned packages :thinking:

Reply:
I also prefer `rand` to `v2`, but I think Go indeed does this by default, so the alias isn't necessary: https://go.dev/play/p/Esxcqx8uaGR

Reply:
Thanks for checking. I'll drop the alias since this seems to be the common pattern e.g. in the Go standard libary itself as well.